ui/app/styles/components/console-ui-panel.scss

```diff
@@ -3,14 +3,15 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+$console-close-height: 35px;
+
 .console-ui-panel {
   background: var(--token-color-palette-neutral-700);
-  width: 100%;
+  width: inherit;
   height: 0;
   min-height: 0;
   overflow: auto;
-  position: absolute;
-  right: 0;
+  position: fixed;
```

Comment:
Absolute was not correct here since the panel may be pushed down out of view when toggled if the page was overflowing.